packages/generator/src/sdk-metadata/__snapshots__/sdk-metadata.spec.ts.snap

```diff
@@ -6,52 +6,57 @@ exports[`sdk-metadata generates the sdk metadata content for services with prege
 Object {
   "generationAndUsage": Object {
     "apiSpecificUsage": Object {
-      "header": "Sample code for using the client library",
+      "header": "Usage Example",
       "instructions": "import { TestEntity } from '@sap/cloud-sdk-vdm-test-service';
 
 const resultPromise = TestEntity.requestBuilder().getAll().top(5).execute({ destinationName:'myDestinationName' });
 ",
-      "text": "In the example below you query the top 5 entities of the  <code>TestEntity </code> type.",
+      "text": "To consume the service via the pregenerated typed client library run the code snippet below. For more details about OData client libraries chose \\"OData Consumption Manual\\" from the \\"Helpful Links\\" menu.",
     },
     "generationSteps": Object {
+      "header": "Installation & Generation Steps",
       "instructions": "<ul>
-<li>Download the API specification to your local computer.</li>
-<li>Install the generator <code>npm install -g @sap-cloud-sdk/generator</code></li>
+<li>Install <a href=\\"https://docs.npmjs.com/downloading-and-installing-node-js-and-npm\\">Node and npm.</a>. The use of the LTS version is preferred.</li>
+<li>Download the API specification from Overview -> API Specification section. Chose EDMX format.</li>
+<li>Generate a typed client by running <code>npm install -g @sap-cloud-sdk/generator</code> to install the code generator.</li>
 <li>Execute the generator <code>generate-odata-client --inputDir path/to/service-spec --outputDir path/to/</code></li>
-</ul>
-The steps above will generate a basic TypeScript client. For additional options like transpiling to JavaScript etc. visit our <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client\\">documentation.</a>",
-      "text": "Follow the following generation steps to generate the client.",
+<li>You should now find generated classes in the output directory specified in step 4.</li>
+<li>By default we generate TypeScript only. If you need to transpile it to JavaScript check the <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client#options>\\"code generator CLI parameters in the documentation.</a></li>
+<li>In case of any issues carefully check your configuration and refer to our <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client\\">extended typed client generation manual.</a></li>
```

Comment:
```suggestion
<li>In case of any issues, carefully check your configuration and refer to our <a href=\\"https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client\\">extended typed client generation manual</a>.</li>
```